src/EA.Weee.DataAccess.Tests.DataAccess/StoredProcedure/GetAatfEvidenceSummaryTotalsTests.cs

```diff
@@ -0,0 +1,360 @@
+﻿namespace EA.Weee.DataAccess.Tests.DataAccess.StoredProcedure
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using AutoFixture;
+    using Core.Helpers;
+    using Domain.AatfReturn;
+    using Domain.Evidence;
+    using Domain.Lookup;
+    using EA.Weee.Tests.Core;
+    using EA.Weee.Tests.Core.Model;
+    using FluentAssertions;
+    using Weee.DataAccess.StoredProcedure;
+    using Xunit;
+    using Organisation = Domain.Organisation.Organisation;
+
+    public class GetAatfEvidenceSummaryTotalsTests
+    {
+        private readonly Fixture fixture;
+
+        public GetAatfEvidenceSummaryTotalsTests()
+        {
+            fixture = new Fixture();
+        }
+
+        [Fact]
+        public async Task Execute_GivenAatfWithDraftNote_NoDataShouldBeReturned()
```

Comment:
Could the first three tests be replaced with a [Theory] With something like "GivenAatfWithNonApprovedNote_NoDataShouldBeReturned(NoteStatus status)"

Reply:
Good idea, have updated.